src/core_plugins/kbn_vislib_vis_types/public/histogram.js

```diff
@@ -29,6 +29,7 @@ export default function HistogramVisType(Private) {
       modes: ['stacked', 'percentage', 'grouped'],
       editor: histogramTemplate
     },
+    isLegendCountSupported: true,
```

Comment:
In fact, I just tested this PR and I'm pretty sure this is not the right place for this parameter.  [`$scope.vis.type.isLegendCountSupported`](https://github.com/elastic/kibana/pull/7970/commits/66bb668a985691bc34f116087aae32319bd699c0#diff-004919223b45b7fa1809b77ad6ab4851R71) is evaluating to `undefined`.